Swiftification.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Swiftification"
-  s.version      = "8.0.0"
+  s.version      = "9.0.0"
```

Comment:
We could avoid that by keeping the old version and marking it deprecated, would you prefer that?